stable/app/templates/cron.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.cron.enabled }}
+{{- range $job := .Values.cron.jobs }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: "{{ include "app.fullname" . }}"
```

Comment:
How would this work if we have multiple cron jobs defined?
Wouldn't same name have clashes?

Reply:
yes. there would be a name clash.  updated the name value to take in app-name and job-name both concatenated.